test/functional/rpc_blockchain.py

```diff
@@ -142,11 +140,11 @@ def _test_getblockchaininfo(self):
         assert_greater_than(res['size_on_disk'], 0)
 
         assert_equal(res['softforks'], {
-            'bip34': {'type': 'buried', 'active': True, 'height': 2},
-            'bip66': {'type': 'buried', 'active': True, 'height': DERSIG_HEIGHT},
-            'bip65': {'type': 'buried', 'active': True, 'height': CLTV_HEIGHT},
-            'csv': {'type': 'buried', 'active': False, 'height': 432},
-            'segwit': {'type': 'buried', 'active': True, 'height': 0},
+            'bip34': {'type': 'buried', 'active': True, 'height': 1},
```

Comment:
Wouldn't having slightly different activation heights here (even it its 1,2,3,4,5) make for a better test?

Reply:
Can do on the next force push or another pull request (whichever happens earlier)